<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1363</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1311</width>
      <height>661</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLine"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Surname:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="surnameLine"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Age:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ageLine"/>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Index of row:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="indexLine"/>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Index of row:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="indexLine_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLine_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Surname:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="surnameLine_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Age:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ageLine_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="editButton">
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
